lib/AST/ASTImporter.cpp

```diff
@@ -2453,10 +2455,11 @@ ExpectedDecl ASTNodeImporter::VisitEnumDecl(EnumDecl *D) {
 
   SourceLocation ToBeginLoc;
   NestedNameSpecifierLoc ToQualifierLoc;
+  SourceRange ToBraceRange;
   QualType ToIntegerType;
-  if (auto Imp = importSeq(
-      D->getBeginLoc(), D->getQualifierLoc(), D->getIntegerType()))
-    std::tie(ToBeginLoc, ToQualifierLoc, ToIntegerType) = *Imp;
+  if (auto Imp = importSeq(D->getBeginLoc(), D->getQualifierLoc(),
```

Comment:
Do we still need this hunk of change in this PR? Isn't that handled in #633 ?

Reply:
This part will be removed (by rebase) after #632 is applied.

Reply:
I see, but it would make sense to create the PRs independently. I mean it would be nice if it would possible to apply #633 independently of #632. That would mean we can merge #633 first. Also upstreaming would be more straightforward.

Reply:
#633 can be applied first, then update this PR with force-push to remove unneeded parts.

Reply:
OK